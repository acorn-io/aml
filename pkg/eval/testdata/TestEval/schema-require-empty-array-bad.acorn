define Foo: {
	a: array
}

Foo(
{}
)