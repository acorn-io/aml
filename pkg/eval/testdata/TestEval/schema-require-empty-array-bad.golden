"missing required key(s): [.a] (schema-require-empty-array-bad.acorn:1:8)"
